TrackingTools/GsfTools/src/MultiTrajectoryStateMode.cc

```diff
@@ -11,9 +11,10 @@
 
 #include <iostream>
 
+namespace MultiTrajectoryStateMode {
```

Comment:
cms coding rules require namespaces to start with a lowercase...
please ```sed -i 's/MultiTrajectoryStateMode/multiTrajectoryStateMode/g' ``` or equivalent